tests/phpunit/TemplateTest.php

```diff
@@ -182,6 +182,10 @@ public function testMisspeltParameters() {
     $this->assertNotNull($expanded->get('journal'));
     $this->assertNotNull($expanded->get('pages'));
     $this->assertNotNull($expanded->get('year'));
+      
+    $text = "{{Cite journal|pp. 1–5}}";
```

Comment:
Is this redundant to `testUseUnusedData` (line 36)?

Reply:
This is for that bug where the first element gets deleted.  Don't want it coming back